styles/main.css

```diff
@@ -0,0 +1,189 @@
+
+section.trips {
+  margin-top: 1rem;
+}
+
+body > main > section.trips > ul {
+  list-style: none;
+  margin-left: 0;
+}
+
+section.trips {
+  margin: 1.5rem 1rem;
+}
+
+.trip img {
+  width: 100%;
+  height: auto;
+  margin-bottom: 0;
+}
+
+.trip > div {
+  margin-bottom: 2rem;
+}
+
+.trip p.continent {
+  text-align: right;
+  padding-right: .75rem;
+}
+.trip p.weeks {
+  text-align: left;
+  padding-left: .75rem;
+}
+
+.trip-name {
+  font-size: 1.5rem;
+  font-family: "Kanit", sans-serif;
+  color: #075523;
+  text-align: center;
+  padding-left: 1rem;
+  padding-right: 1rem;
+}
+
+body.init {
+  background-image: url('./images/mts-cow.jpeg');
+  background-size: cover;
+  background-position: center;
+  height: 100vh;
+}
```

Comment:
I'd love to know what you think about some variation of the following style to help improve readability of the white text:
```css
body.init {
  background: linear-gradient(
      rgba(0, 0, 0, 0.4),
      rgba(0, 0, 0, 0.4)
    ), url('./images/mts-cow.jpeg');
  background-size: cover;
  background-position: center;
  height: 100vh;
}
```